students/josh_embury/lesson04/dict_lab.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+#Dictionaries 1:
+# create dictionary:
+dict1 = {'name':'Chris', 'city':'Seattle','cake':'chocolate'}
+#display dictionary:
+for val in dict1.items():
+    print(val)
+#delete entry:
+del dict1['cake']
+#display dictionary:
+print()
+for val in dict1.items():
+    print(val)
+#add entry
+dict1['fruit'] = 'Mango'
+# display keys
+print('\nKeys in dict:')
+for keys in dict1.keys():
+    print(keys)
+# display values
+print('\nValues in dict:')
+for vals in dict1.values():
+    print(vals)
+print('\nIs cake in dict?')
+print('cake' in dict1)
+print('\nIs mango in dict?')
+print('Mango' in dict1.values())
+
+#Dictionaries 2:
+dict2 = {}
+for dicval in dict1:
+    dict2[dicval] = dict1[dicval].lower().count('t')
+print('Creating new dict with t count:')
+for val in dict2.items():
+    print(val)
+print()
+#Sets
+s2=set()
```

Comment:
Need spaces here and below too, around assignment operator

Reply:
updated